components/automate-cs-oc-erchef/habitat/config/sys.config

```diff
@@ -162,8 +162,10 @@
 
     {chef_index, [
         {ip_mode, [{{cfg.ip_mode}}]},
-        {search_provider, elasticsearch},
-        {search_queue_mode, batch},
+        {search_provider, {{cfg.index.search_provider}}}, 
+        {search_queue_mode, {{cfg.index.search_queue_mode}}},
+        {search_auth_username, "{{cfg.index.search_auth_username}}"},
+        {search_auth_password, "{{cfg.index.search_auth_password}}"},
```

Comment:
Do the quotes around the values for search_auth_username and search_auth_password need to be removed?